packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -261,9 +265,14 @@ Only Okta manages standard resource sets, so you can't update or delete them.
 | `okta.devices.lifecycle.delete` <br><ApiLifecycle access="ea" />                              | Allows the admin to permanently delete devices                                                                                                 | All devices
 | `okta.devices.read` <br><ApiLifecycle access="ea" />                              | Allows the admin to read device details                                                                                                 | All devices |
 | `okta.iam.read` | Allows the admin to view roles, resources, and admin assignments                                                                  | All Identity and access management resources |
+| `okta.realm.manage` | Allows the admin to view, create, and manage realms | All realm resources |
```

Comment:
We should mark these with EA tag?

Reply:
I added this here for completeness, but it's not necessary since this entire table is hidden and will probably be removed in the future. We're going to point everyone to the Permissions page on Redocly (so that we only have to update one source)